addons/common/functions/fnc_headBugFix.sqf

```diff
@@ -10,21 +10,30 @@
 
 #include "script_component.hpp"
 private ["_pos","_dir","_anim"];
-if (player != vehicle player  || {(player getvariable ["ace_isUnconscious", false])}) exitWith {};
+
+_anim = animationState ACE_player;
+["HeadbugFixUsed", [profileName, (animationState ACE_player)]] call FUNC(serverEvent);
+["HeadbugFixUsed", [profileName, (animationState ACE_player)]] call FUNC(localEvent);
+
+if (ACE_player != vehicle ACE_player  || { !([ACE_player, objNull, ["isNotSitting"]] call FUNC(canInteractWith)) } ) exitWith {false};
+
+_pos = getposATL ACE_player;
+_dir = getDir ACE_player;
+
 titleCut ["", "BLACK"];
-_pos = getposATL player;
-_dir = getDir player;
-_anim = animationState player;
+
 // create invisible headbug fix vehicle
-_ACE_HeadbugFix = createVehicle ["ACE_Headbug_Fix", getposATL player, [], 0, "NONE"];
+_ACE_HeadbugFix = createVehicle ["ACE_Headbug_Fix", [0,0,10000], [], 0, "NONE"];
```

Comment:
Is it possible to do this with only a local vehicle? (Just to save on the network traffic)